extmod/mbedtls/mbedtls_alt.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
This file somewhat follows code from `ecdsa.c` which is Apache/GPL.  Let me know if the license is okay.

Reply:
This file could contain multiple other wrappers. I've added a line just above the wrapper function if it helps:
// See lib/mbedtls/library/ecdsa.c:688